src/renderer.js

```diff
@@ -120,7 +135,11 @@ export default function Emoji({
     return true;
   }
 
-  const output = toArray(text, options);
+  let output = returnStrippedElements(
```

Comment:
How about naming the function `returnNonStringStrippedElements` ? I would say that's a more accurate description of the function's logic and it matches better the opposite of the `stripNonStringElements` function.